### PR TITLE
HDDS-5747. Reset default values in ContainerBalancerConfiguration

### PR DESCRIPTION
**Pull request by siddhantsangwan**
## What changes were proposed in this pull request?

Currently, the default values for `maxSizeEnteringTarget` and `maxSizeLeavingSource` are 5GB. These should be reset to Container size + 1GB, so that at least one container is always available to be selected for move.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-5747

## How was this patch tested?

Added a test in `TestContainerBalancer` UT.

### CODE REVIEW

**Comment by siddhantsangwan** [comment]
@lokeshj1703 @JacksonYao287 please take a look.

**Review by JacksonYao287 — APPROVED** [review]
thanks @siddhantsangwan for this work!  LGTM +1

File: hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java [hunk 1]
```diff
@@ -61,25 +63,29 @@
...
-  private long maxSizeToMovePerIteration = 10 * OzoneConsts.GB;
+  private long maxSizeToMovePerIteration = 30 * OzoneConsts.GB;
 
   @Config(key = "size.entering.target.max", type = ConfigType.SIZE,
-      defaultValue = "5GB", tags = {ConfigTag.BALANCER}, description = "The " +
-      "maximum size that can enter a target datanode while balancing. This is" +
-      " the sum of data from multiple sources.")
-  private long maxSizeEnteringTarget = 5 * OzoneConsts.GB;
+      defaultValue = "", tags = {ConfigTag.BALANCER}, description = "The " +
+      "maximum size in Gigabytes that can enter a target datanode in each " +
```
Comment (lokeshj1703) [inline hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java:76]: NIT. Let's remove `in Gigabytes`.

Comment (lokeshj1703) [reply hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java:76]: This is mentioned in both the configs.

**Comment by siddhantsangwan** [comment]
Thanks for the reviews! I've addressed them.

**Review by lokeshj1703 — APPROVED** [review]
@siddhantsangwan Thanks for updating the PR! The changes look good to me. +1.

**Pull request merged**